app/model/entity/Login.php

```diff
@@ -94,6 +94,14 @@ public function passwordsMatch($password) {
     return FALSE;
   }
 
+  /**
+   * Factory method.
```

Comment:
Does this docblock convey any useful information?

Reply:
Just for satisfying PhpStorm...

Reply:
I can tolerate its existence, but you might want to consider turning that inspection off.